### ISSUE TITLE
can't find the file 'dtw_mat.txt'

### ISSUE THREAD

**Issue by keliang12138** [Jackxiini/LB-SimTSC#1]
when the lb=0,  distances = read_dtw_mt_from_txt(ucr_dtw_mt_dir+'/'+args.dataset+'/dtw_mat.txt')  can't find the file 'dtw_mat.txt'

**Comment by Jackxiini** [comment]
Please run MATLAB code for generating dtw_mat.txt if lb=0

**Comment by keliang12138** [comment]
Dear author, may I ask that the sample number of my data set is 12000, the sequence length is 2048, the training set and the test set are 6000x2048, but I have been running the matlab code you provided for a week, and the running has not finished, is there something wrong?

**Comment by Jackxiini** [comment]
Running time may depend on your machine and the size of your dataset. For very large dataset, computing the DTW matrix could take a long time. You can subset the dataset to check if there is a problem.
